app/views/searches/new.html.erb

```diff
@@ -0,0 +1,17 @@
+<h1>Search Results (<%= @search_type %>)</h1>
+<% if @students && !@students.empty? %>
+  <ul>
+  <% @students.each do |student| %>
+    <li><%= link_to student.full_name, student_path(student) %></li>
+  <% end %>
+  </ul>
+<% elsif @labs && !@labs.empty? %>
+  <ul>
+  <% @labs.each do |lab| %>
+    <li><%= link_to lab.name, [lab.cohort, lab] %></li>
+  <% end %>
+  </ul>
+<% else %>
+  <p>No <%= @search_type %> matching your query. Please revise your search term.</p>
+<% end %>
+
```

Comment:
instead, we'll be operating on search or search result instance. So, logic around student/lab results should be refactored into instance method on search class.